### ISSUE TITLE
Annotation errors in relative clauses

### ISSUE THREAD

**Issue by xiulinyang** [UniversalDependencies/UD_English-EWT#503]
### The following sentences are not relative clauses but the words in bold are marked with _acl:relcl_.
- It marks the first time in recent years that China has promised to wield its veto power in the UN Security Council against a petition **initiated** by the United States and backed by France and Great Britain."
- Correspondents who flocked in to cover the tsunami found they also had another story - the miserable security situation surrounding the Indian navy's Far Eastern Naval Command, now **being established** along with India's "blue water navy".
- At some point, I'd like to talk about the diversification strategy in more detail -- perhaps over dinner or a quick meeting after **the markets close**?
- His explosive temper probably provoked the disastrous siege of Fallujah last spring, killing 600 Iraqis, most of them **women and children**, in revenge for the deaths of 4 civilian mercenaries, one of them a South African
- "Calculation of Floating Amount: The Floating Amount **payable** by [ENA] on a Payment Date will be calculated for that Payment Date as follows:
- Please find enclosed EES's request for Volumes for PGE CityGate delvery **effective** 11/1/01:
- Because of an international medical convention **ongoing** in Washington at the same time as our meetings, the Grand Hyatt unfortunately is already booked.
- The best there **is in service**
- The mysteries of the universe continue to attract and intrigue you, as you apply your mystical awareness to the important decisions **upcoming i**n your life, especially concerning partnerships of all kinds.
- Given the chance **your cat may only have one kidney,** I would think the worst case scenario would be kidney disease, which untreated can lead to possible renal failure.
- Your application is still evaluated on a number of criteria, not all of them as "**cast in steel"** as the points.
In this day its rare to find such wonderful people who CARE , Not the kind of want to make cash.
- THEN put the eggs in water **treated** a couple of days before with a water treatment and left open to "breathe".

### dependency relations are wrong for the tokens in bold.
- A country deserves the leaders it has, **my friends**
- There must be a reason so many people like **it** there.
- Mike one of owners was awesome, he explained the detailed plan, and executed on time, I am always going use them and refer them to many friends I can because of the great job they did me.

### not sure if they are correct or not/sentences are ungrammatical
- The dangers implicit in any attack upon that nation, however, seem to significantly offset **whatever** gains could be made in the so-called "War on Terror."
- Overall they aren't very knowledge about the type of games **are on the market.**
- Your application is still evaluated on a number of criteria, not all of them as "cast in steel" as the points.
In this day its rare to find such wonderful people who CARE , **Not the kind of want to make cash.**

**Comment by nschneid** [comment]
Thanks! Agree on most of the ones that should be `acl`.

> * The best there **is in service**

This one is tricky. "The best there is" is an idiom. I think it is a relative clause, and "best" stands for roughly "the best option". It's hard to make a paraphrase with "there is" that preserves the semantics/information structure though.

> * There must be a reason so many people like **it** there.

I think this is correct as expl. "It" is nonreferential.
 
> * Mike one of owners was awesome, he explained the detailed plan, and executed on time, I am always going use them and refer them to many friends I can because of the great job they did me.

Where is the issue here?

> * The dangers implicit in any attack upon that nation, however, seem to significantly offset **whatever** gains could be made in the so-called "War on Terror."

"whatever gains could be made (in the...)" is a free relative. The only issue I see in the basic tree is that `PronType=Int` should be `Rel`.
 
> * Overall they aren't very knowledge about the type of games **are on the market.**

Yeah, a relativizer is missing.

> * Your application is still evaluated on a number of criteria, not all of them as "cast in steel" as the points.

Grammatical ("cast in steel" is being used like an adjective: cf. "not all of them as rigid as the points" or "not all of which are as rigid as the points"). Will have to think about this but my hunch is it should be `acl` not `acl:relcl`.

>   In this day its rare to find such wonderful people who CARE , **Not the kind of want to make cash**

Yeah this is ungrammatical. Maybe "of" was supposed to be "who"? "of" is attached as `dep` suggesting incoherence.

**Comment by xiulinyang** [comment]
Thanks for the clarification! I think _me_ in the sentence below should be _iobj_?
> Mike one of owners was awesome, he explained the detailed plan, and executed on time, I am always going use them and refer them to many friends I can because of the great job they did **me**.

**Comment by nschneid** [comment]
Yes, I would agree with the iobj annotation though it sounds a bit nonstandard to my ear.

**Comment by nschneid** [comment]
Most of the ones that should be `acl` can be identified because they are non-finite clauses, or are finite but marked as a whole by a word like "after". Generalizing to queries:

- [not finite](https://universal.grew.fr/?custom=658f2a729eb45): most should be `acl`, though some are actually finite (tagging error on the verb, or omitted finite aux that is understood)
- [unexpected marker](https://universal.grew.fr/?custom=658f2ba2ec392): 2 are "after" clauses that should be `acl`, one is "on which to" which can remain as `acl:relcl`, and one is an anomalous sentence

**Comment by nschneid** [comment]
("the chance that" and vocative "my friends" were addressed in 495a62f)

**Comment by nschneid** [comment]
Ones I'm not sure about—@amir-zeldes? 

- killing 600 Iraqis, **most of them women and children**: analyzed as a verbless clause. `acl` or `parataxis`?
- a number of criteria, **not all of them as "cast in steel" as the points**: same
- for customers **already on DA**: does "already" signal that this is a clause, hence `acl`, or could it be `nmod` with an `advmod`?

**Comment by amir-zeldes** [comment]
I think `parataxis` - the use of a PRP in "a number of criteria, not all of them as "cast in steel" as the points: same" seems telling, since English generally avoids subj/obj resumptives in acls.

customers already on DA: would go with nmod, same as "customers on DA" plus an adjunct.

Ongoing etc. look like extraposed adj, like "a table similar to a chair", so +1

**Comment by nschneid** [comment]
> I think `parataxis` - the use of a PRP in "a number of criteria, not all of them as "cast in steel" as the points: same" seems telling, since English generally avoids subj/obj resumptives in acls.

I guess it depends which acls we're talking about...a lot of them are noun complements, for example, but I don't see that as particularly relevant to this construction.

Looking at the verbless clauses ([GUM](https://universal.grew.fr/?custom=65903acc99364), [EWT](https://universal.grew.fr/?custom=65903b5003974)), it looks like GUM puts several verbless clauses with "each"-subjects under `acl` (which seems like the same general construction minus the explicit partitive *of*-phrase). E.g.

- competing worlds, **each with their own dominant currency**: "each of them" is understood (implicit partitive)

"Some of them very touristic" is the only `parataxis` one of this kind in GUM. The other main verbless `acl` construction in both treebanks is the *with*-absolute construction, which seems like a cousin of the one we're deciding.

Overall, among the verbless clauses, the ones under `acl` are more closely integrated with the head nominal than the `parataxis` ones. I'm not sure we want to make an exception to that based on pronouns.

**Comment by amir-zeldes** [comment]
Agreed, it should be uniform, and my tendency looking at the examples is to change acl to parataxis as parenthetical, except maybe the ones introduced by "with", since I think that's a kind of complementizer that suggests a real adjunct structure (prepositional, just mark because its arg is a clause).

Would you be on board with `acl` for "with X Y" and `parataxis` otherwise? It also gels with the `advcl` reading of "sit with your legs crossed" in #240 .

**Comment by nschneid** [comment]
OK. Now all verbless acls with subjects are _with_-absolutes [in EWT](https://universal.grew.fr/?custom=6591887109350).

**Comment by nschneid** [comment]
Actually, I just realized that this construction has a relative clause counterpart ([GUM](https://universal.grew.fr/?custom=6592f71b97595), [EWT](https://universal.grew.fr/?custom=6592f7390556c)). 

Taking the above example:

- killing 600 Iraqis, most of **them** women and children: `???(Iraqis, women)`

We can rephrase as:

- killing 600 Iraqis, most of **whom were** women and children: `acl:relcl(Iraqis, women)`

I think the parallelism is a good reason to go with `acl` for the first one over `parataxis`.

**Comment by amir-zeldes** [comment]
I'm not sure about that... I mean, sure, a relative paraphrase for that exists, but that's probably true of a lot of parataxes. What exactly is the criterion to make this one be acl? What I like about singling out "with" is that the criterion is fairly clear: it's an explicit complementizer, which realizes explicit syntactic subordination. I can't say that about things like:

  * I like Bob (he's my cousin) because he's a nice guy

There is a paraphrase:

  * I like Bob, who is my cousin, because he's a nice guy

And the latter is definitely a subordinating construction. But in the former example "(he's my cousin)" is what I'd call a paradigm example of a parenthetical, which should therefore be `parataxis`.

**Comment by nschneid** [comment]
"He's my cousin" can stand as an independent clause, so that's clearly parataxis. It seems to me that the verbless construction we're talking about is used more narrowly to expand on a nominal.

**Comment by nschneid** [comment]
Relatedly, verbless absolute constructions can occur in a clause with or without "with": "(With) his hands in the air, the suspect was arrested." Those are both advcl right? I think this shows that subject+verbless predicate constructions tend to be modifiers as opposed to parataxis.

**Comment by amir-zeldes** [comment]
> "He's my cousin" can stand as an independent clause

What about:

  * I like Bob (formerly my boss) because he's a nice guy

I think that can't stand alone - is that not a parenthetical parataxis?

**Comment by nschneid** [comment]
I would say "Bob (my boss)" is appos. Is it appos with "formerly"? I guess it depends how strictly the reversibility criterion is enforced. If it's not appos then it's parataxis by default—not a clause right?

**Comment by nschneid** [comment]
To expand on the problem, below we have a range of paraphrases that tack on an elaboration about the applicants:

1. Each year the program admits a dozen applicants, mostly international students.
2. Each year the program admits a dozen applicants, most of them international students.
3. Each year the program admits a dozen applicants, most of which are international students. - clearly a relative clause
4. Each year the program admits a dozen applicants, most of them coming from outside the U.S.
5. Each year the program admits a dozen applicants, mostly coming from outside the U.S.

How should lines be drawn here? It's not obvious to me.

For elaborating clauses, @amir-zeldes suggests explicit subordination (with a word like "with" or "if" or a relative clause) as a test for modification status (`acl` or `advcl`), with `parataxis` coming into play otherwise.

It seems to me that there is a construction—call it the Partitive Subject Elaboration construction—that (2-4) share in common. Maybe this is reason enough to consider them all some version of `acl`, or maybe the Partitive Subject Elaboration construction is orthogonal to the modification-vs-parataxis distinction in UD. Interestingly, a partitive subject depictive variant of (4) with an explicit verb could occur sentence-initially ("Most of the applicants coming from outside the U.S., a dozen are admitted each year") but I don't think that works with the verbless clause in (2) ("*Most of the applicants international students, a dozen are admitted each year"), and it certainly doesn't work with the relative clause in (3).

**Comment by amir-zeldes** [comment]
> It seems to me that there is a construction—call it the Partitive Subject Elaboration construction—that (2-4) share in common. 

1. is an apposition, and 5. is an adnominal gerund clause (`acl`) so I agree those are separate. I can see 2 and 4 as being the same, whatever we call them, but 3. is a relative clause, which is a different, well understood construction, and uniquely `acl:relcl`.

**Comment by nschneid** [comment]
> 1. is an apposition

Wouldn't (1) be like "formerly my boss"? I thought you said no to `appos` for that because of the adverb?

To add another one to the mix:

6\. Each year the program admits a dozen applicants, mostly from outside the U.S.

**Comment by amir-zeldes** [comment]
Oh I see... yes, OK, maybe that's also not revsersible, I wasn't sure. In any case, I think 3. is totally different, and 2. is probably a verbless version of 4., so I agree they should ideally be the same.

**Comment by nschneid** [comment]
> Oh I see... yes, OK, maybe that's also not revsersible, I wasn't sure.

OK to be fair this is an acceptable sentence:

- Each year the program admits mostly international students (a dozen applicants).

But the meaning changes: I would take this to mean that all of the dozen are international, and "mostly" is in reference to some understood larger superset.
